python/ql/test/library-tests/regex/options

```diff
@@ -1 +1 @@
-semmle-extractor-options: --max-import-depth=3
```

Comment:
Since max-import-depth=1 is the [default](https://github.com/github/codeql/blob/main/python/ql/test/library-tests/options),  can we remove this file now please? :)

Reply:
Done.